nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateJson.java

```diff
@@ -79,65 +92,69 @@
         }
 )
 public class ValidateJson extends AbstractProcessor {
-    public enum SchemaVersion implements DescribedValue {
-        DRAFT_4("Draft Version 4", "Draft 4", VersionFlag.V4),
-        DRAFT_6("Draft Version 6", "Draft 6", VersionFlag.V6),
-        DRAFT_7("Draft Version 7", "Draft 7", VersionFlag.V7),
-        DRAFT_2019_09("Draft Version 2019-09", "Draft 2019-09", VersionFlag.V201909),
-        DRAFT_2020_12("Draft Version 2020-12", "Draft 2020-12", VersionFlag.V202012);
-
-        private final String description;
-        private final String displayName;
-        private final VersionFlag versionFlag;
-
-        SchemaVersion(String description, String displayName, VersionFlag versionFlag) {
-            this.description = description;
-            this.displayName = displayName;
-            this.versionFlag = versionFlag;
-        }
 
-        @Override
-        public String getValue() {
-            return name();
-        }
+    public static final String SCHEMA_NAME_PROPERTY_NAME = "Schema Name";
+    public static final String SCHEMA_CONTENT_PROPERTY_NAME = "JSON Schema";
+    public static final String ERROR_ATTRIBUTE_KEY = "json.validation.errors";
+    public static final AllowableValue SCHEMA_NAME_PROPERTY = new AllowableValue("schema-name", "Use '" + SCHEMA_NAME_PROPERTY_NAME + "' Property",
+            "The name of the Schema to use is specified by the '" + SCHEMA_NAME_PROPERTY_NAME +
+                    "' Property. The value of this property is used to lookup the Schema in the configured JSON Schema Registry service.");
+    public static final AllowableValue SCHEMA_CONTENT_PROPERTY = new AllowableValue("schema-content-property", "Use '" + SCHEMA_CONTENT_PROPERTY_NAME + "' Property",
+            "A URL or file path to the JSON schema or the actual JSON schema is specified by the '" + SCHEMA_CONTENT_PROPERTY_NAME + "' Property. " +
+                    "No matter how the JSON schema is specified, it must be a valid JSON schema");
 
-        @Override
-        public String getDisplayName() {
-            return displayName;
-        }
+    private static final List<AllowableValue> STRATEGY_LIST = Arrays.asList(SCHEMA_NAME_PROPERTY, SCHEMA_CONTENT_PROPERTY);
 
-        @Override
-        public String getDescription() {
-            return description;
-        }
+    public static final PropertyDescriptor SCHEMA_ACCESS_STRATEGY = new PropertyDescriptor.Builder()
+            .name("schema-access-strategy")
+            .displayName("Schema Access Strategy")
```

Comment:
For new property values, it is best to keep the `name` and `displayName` the same.
```suggestion
            .name("Schema Access Strategy")
            .displayName("Schema Access Strategy")
```